Automattic/pocket-casts-android
# Add "Change Avatar" button to account settings

## Description

Adding the ability to change the avatar. Avatar is coming from user's Gravatar account. So the "Change Avatar" button opens the Gravatar web link via browser. Similar to what we do with `Privacy policy` or `Terms of use`.

cc: @mlumeau @AdamGrzybkowski 

## Testing Instructions

1. Enable `GRAVATAR_CHANGE_AVATAR` feature flag under Profile -> Settings -> Beta features
2. Tap on the `Profile` tab
3. Tap on `Change avatar`
4. Verify Gravatar is opened in the browser
5. Modify your avatar
6. Return to PocketCast
7. Verify you can see the new avatar
8. Disable `GRAVATAR_CHANGE_AVATAR` feature flag under Profile -> Settings -> Beta features
9. Tap on the `Profile` tab
10. Verify you can't see the `Change avatar` row

## Screenshots or Screencast 
<img width="250" alt="image" src="https://github.com/Automattic/pocket-casts-android/assets/6974554/ff316e5e-2367-4588-9e68-ebfd886bcda3">

## Checklist
 
#### I have tested any UI changes...
